tobac/themes/tint/tracks.py

```diff
@@ -72,8 +72,12 @@ def make_tracks(grid_ds, field, params=None):
         record.add_uids(current_objects)
         tracks = write_tracks(
             tracks, record, current_objects, obj_props)
-        cell_mask[i] = frame1
-         
+        uids = np.array([int(x) for x in current_objects['uid']])
```

Comment:
this all looks good, but without testing capabilities it is hard to tell more ;-)